app/controllers/application_controller.rb

```diff
@@ -1,5 +1,11 @@
-class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.first
-  end
-end
+class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  before_action :update_allowed_parameters, if: :devise_controller?
+
```

Comment:
- [ ] All pages should be secured so that a non-logged-in user cannot access them and the user should be redirected to the sign-in path. Kindly add the code below to fix this.

  ```
  before_action :authenticate_user!
  ```